tests/data/mtx/mtx_character_before_data.mtx

```diff
@@ -0,0 +1,8 @@
+%%MatrixMarket matrix coordinate real general
+a
```

Comment:
This line is invalid for the file format and should trigger the ValueError ""MTX file did not contain expression data"